ShestakUI/Modules/Cooldowns/EnemyCD.lua

```diff
@@ -102,13 +102,15 @@ local CreateIcon = function()
 	return icon
 end
 
-local StartTimer = function(name, sID)
+local StartTimer = function(sGUID, sID)
 	local _, _, texture = GetSpellInfo(sID)
 	local icon = CreateIcon()
 	icon.Texture:SetTexture(texture)
 	icon.Texture:SetTexCoord(0.1, 0.9, 0.1, 0.9)
 	icon.endTime = GetTime() + T.EnemySpells[sID]
-	local color = (CUSTOM_CLASS_COLORS or RAID_CLASS_COLORS)[select(2, UnitClass(name))]
+	local locClass = "UNKNOWN"
+	locClass, _, _, _, _, name, _ = GetPlayerInfoByGUID(sGUID)
+	local color = (CUSTOM_CLASS_COLORS or RAID_CLASS_COLORS)[string.upper(locClass):gsub("%s+", "")]
```

Comment:
I think, this better
local _, class, _, _, _, name = GetPlayerInfoByGUID(sGUID)
local color = (CUSTOM_CLASS_COLORS or RAID_CLASS_COLORS)[class]

Reply:
Yeah, it's way better I don't know how I mess this obvious possibility, I remember to tried many things to avoid all process on the string... Sorry for such a mistake...